src/containers/redditClone/commentPage.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { addNewComment, updateComment, getCommentPagePost, getPostId } from 'redux/actions/comments'
+import CommentPage from 'components/redditClone/commentPage/index'
+
+export class CommentPageContainer extends Component {
+  componentDidMount() {
+    this.props.dispatch(getPostId(this.props.params.postId))
+    this.props.dispatch(getCommentPagePost())
```

Comment:
Idea: Does it make sense to consolidate these two dispatches into one.

e.g. `getCommentPagePost(this.props.params.postId)` 

or is `getPostId` used elsewhere which I'm missing?